test/tutorial/repo/load_repo.cpp

```diff
@@ -1,22 +1,22 @@
-// create a reference to the Base's repo_sack for better code readability
-auto repo_sack = base.get_repo_sack();
-
-// create a new repo with a given repoid
-// the repo is a weak pointer to an object owned by the repo_sack
-std::string repoid = "example";
-auto repo = repo_sack->create_repo(repoid);
+// Create a new repo with the given id.
+//
+// The repo is a weak pointer to an object owned by the repo sack.
+auto repo = base.get_repo_sack()->create_repo("my_new_repo_id");
```

Comment:
I somehow do not understand why we have two tutorials for loading repositories. What about to merge them and instead of creating a repository an load it separately to load all repositories using your new functionality? It will help us to ensure that all required logic was performed on the background.

Reply:
As we've discussed, one of them is loading repos from the system config files and the other one demonstrates loading a simple single custom repo. I'd prefer to keep the low-level loading methods in the tutorial for now. It captures the current state of the API, as long as we have those, they need to be valid (and this way the two repo-loading tutorials nicely cover bolth low-level and high-level interface). Once we clearly define the background loading mechanisms for excludes and repo internalizing, we can update the tutorial accordingly. I can add a TODO there if you want though?

On a related note I've pushed an update to some of the tutorials' headings to better describe what they cover.